api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -74,25 +69,56 @@ public void runBeforeEachTest() {
 			adminService.setImplementationIdHttpClient(implementationHttpClient);
 			cacheManager = Context.getRegisteredComponent("apiCacheManager", CacheManager.class);
 		}
-		
+
+	}
+
+	/**
+	 * @see AdministrationService#getSystemInformation()
+	 */
+	@Test
+	public void getSystemInformation_shouldReturnSystemInformation() {
+		Map<String, Map<String, String>> systemInfoMap = Context.getAdministrationService().getSystemInformation();
+		assertEquals(5, systemInfoMap.size());
+	}
+
+	/**
+	 * @see AdministrationService#getSystemInformation()
+	 */
+	@Test
+	public void getSystemInformation_shouldAddAllLoadedModulesInformation() {
+		Module module = new Module("name", "moduleId", "packageName", "author", "description", "version");
+		ModuleFactory.loadModule(module, false);
+		int numberOfModules = ModuleFactory.getLoadedModules().size();
+		Map<String, Map<String, String>> systemInfoMap = Context.getAdministrationService().getSystemInformation();
```

Comment:
can you please add newlines in between, like

```java
int numberOfModules = ModuleFactory.getLoadedModules().size();

 Map<String, Map<String, String>> systemInfoMap = Context.getAdministrationService().getSystemInformation(); 

assertEquals(...
```

This arranges your test in AAA, groups of arrange (what you need to setup your test), act (the class/method under test), assert (what do you expect to happen after calling the method).

It makes your tests more readable since I can easily spot what you are testing which is also why the javadoc is not needed.